core/src/main/resources/hudson/security/Messages.properties

```
@@ -25,7 +25,7 @@ GlobalSecurityConfiguration.Description=Secure Jenkins; define who is allowed to
 HudsonPrivateSecurityRealm.WouldYouLikeToSignUp=This {0} {1} is new to Jenkins. Would you like to sign up?
 LegacyAuthorizationStrategy.DisplayName=Legacy mode
 
-HudsonPrivateSecurityRealm.DisplayName=Jenkins’ own user database
+HudsonPrivateSecurityRealm.DisplayName=Jenkins user database
```

I agree with Stefan in https://github.com/jenkinsci/jenkins/pull/9024#issuecomment-1987146279.
Jenkins is unneeded, I'm not a huge fan of user database though, but I could live with it.

What about one of these?

```suggestion
HudsonPrivateSecurityRealm.DisplayName=Local authentication
```

```suggestion
HudsonPrivateSecurityRealm.DisplayName=Internal
```

```suggestion
HudsonPrivateSecurityRealm.DisplayName=Local
```